jwql/instrument_monitors/common_monitors/edb_telemetry_monitor.py

```diff
@@ -376,22 +376,17 @@
 
 from astropy.stats import sigma_clipped_stats
 from astropy.table import Table
-from astropy.time import Time, TimeDelta
 import astropy.units as u
 from bokeh.embed import components, json_item
 from bokeh.layouts import gridplot
-from bokeh.models import BoxAnnotation, ColumnDataSource, DatetimeTickFormatter, HoverTool, Range1d
+from bokeh.models import ColumnDataSource, DatetimeTickFormatter, HoverTool, Range1d
 from bokeh.models.widgets import Tabs, Panel
 from bokeh.plotting import figure, output_file, save, show
 from bokeh.palettes import Turbo256
 from jwql.database import database_interface
-from jwql.database.database_interface import NIRCamEDBDailyStats, NIRCamEDBBlockStats, \
-    NIRCamEDBTimeIntervalStats, NIRCamEDBEveryChangeStats, NIRISSEDBDailyStats, NIRISSEDBBlockStats, \
-    NIRISSEDBTimeIntervalStats, NIRISSEDBEveryChangeStats, MIRIEDBDailyStats, MIRIEDBBlockStats, \
-    MIRIEDBTimeIntervalStats, MIRIEDBEveryChangeStats, FGSEDBDailyStats, FGSEDBBlockStats, \
-    FGSEDBTimeIntervalStats, FGSEDBEveryChangeStats, NIRSpecEDBDailyStats, NIRSpecEDBBlockStats, \
-    NIRSpecEDBTimeIntervalStats, NIRSpecEDBEveryChangeStats, session, engine
+from jwql.database.database_interface import session, engine
 from jwql.edb import engineering_database as ed
```

Comment:
It is hard to tell from the code - are these imports needed?

Reply:
I would say yes based on the other places we removed them and had to add them back. It's tricky because they are imported but not called directed, but necessary for querying all of our tables. I am going to add them back and add `# ruff: noqa` to the end of the line to ignore it since this behavior is expected.